example.config.json

```diff
@@ -2,15 +2,12 @@
   "githubId": "GITHUB_USER_ID",
   "user": {
     "botname": "GITHUB_USER_ID",
-    "appHost": "http://localhost:7000",
-    "apiServer": "www.freecodecamp.com",
```

Comment:
apiServer is used by the bot for things like `thanks` and should not be removed.